src/components/organisms/Members/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { StyleMembers } from "./styles"
+import MemberInfo, { MemberInfoInterface } from "../../molecules/MemberInfo"
+
+interface MembersListInterface extends MemberInfoInterface {
+  id: string
+}
+
+interface MembersInterface {
+  membersList: MembersListInterface[]
+}
+
+const Members: React.FC<MembersInterface> = (props: MembersInterface) => {
```

Comment:
I mentioned this in my PR for the Nav Menu, but how do you make sure the text (description, role, etc.) are in the right language? Are they being passed as props from another component?

Reply:
- Does `Members` needs to know which language to display? No. So, the component is agnostic about the language. The component needs to only receive props.
- The file `src/pages/index.tsx` is the one how will decide which language to render `useTranslations()` . But, you need to worry about that file now. Storybook is mostly about rendering `dump` components. So, imagine that `Members` will be imported to `src/pages/index.tsx` and `Members` will magically render the correct data. At this phase, you don't need to worry about language, just worry about making the component as dumb/agnostic as possible.